framework/include/fvkernels/FVFluxKernel.h

```diff
@@ -0,0 +1,130 @@
+#pragma once
+
```

Comment:
I considered that.  And I would be okay either way.  I leaned toward `Flux` because that seemed like a term that the general FV community would be more use to.